src/ReinforcementLearningZoo/src/algorithms/policy_gradient/sac.jl

```diff
@@ -40,12 +42,15 @@ end
 - `target_qnetwork2`,
 - `start_policy`,
 - `γ = 0.99f0`,
-- `ρ = 0.995f0`,
+- `τ = 0.005f0`,
```

Comment:
Any special reason why we switch `ρ` to `τ` here?

Reply:
I think what he went for is a standard terminology that is used for soft target updates. I have seen it in a lot of implementations. That is what is used in https://arxiv.org/pdf/2008.10861.pdf as well.

Reply:
@Mobius1D is right. I only change the presentation to match the pseudo-code of the paper.

Reply:
I see. This will be a breaking change. So could you also update https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/master/NEWS.md#reinforcementlearningzoojl and bump the version of https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/blob/c7b66e47e939592045326485b9c89b1486df165d/src/ReinforcementLearningZoo/Project.toml#L4
@pilgrimygy

Reply:
OK, I will update it.

Reply:
You need to create a new section named `ReinforcementLearning.jl@v0.10.0`